src/mixxx.cpp

```diff
@@ -1489,15 +932,19 @@ void MixxxMainWindow::rebootMixxxView() {
 }
 
 bool MixxxMainWindow::loadConfiguredSkin() {
+    // TODO: use std::shared_ptr throughout skin widgets instead of these hacky get() calls
```

Comment:
This would require changes in tons of files throughout the GUI code. Let's leave it for another PR.